springdoc-openapi-common/src/main/java/org/springdoc/core/OpenAPIService.java

```diff
@@ -120,6 +120,11 @@ public class OpenAPIService implements ApplicationContextAware {
 	 */
 	private final Optional<List<OpenApiBuilderCustomizer>> openApiBuilderCustomisers;
 
+	/**
+	 * The server base URL customisers.
+	 */
+	private final Optional<List<ServerBaseUrlCustomizer>> serverBaseUrlCustomisers;
```

Comment:
Weird pattern to use `Optional` for class vars when the code has to check if it's null anyway.